src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const data = req.body;
+      const userId = Number(req.params.id);
+      const groupData = await this.userService.manageGroups(userId, data);
+
+      res.status(200).json({ data: groupData, message: 'manageGroups' });
+    } catch (error) {
+      next(error);
+    }
+  }
```

Comment:
Method Review: Review the addition of the manageGroups method. Ensure it's intended to be in the users.controller.ts file and serves its purpose correctly. If it's related to user-group management, it may be fine; otherwise, it might need to be moved to a more appropriate file.